components/Shell.tsx

```diff
@@ -118,6 +120,10 @@ export default function Shell(props: {
   subtitle?: ReactNode;
   children: ReactNode;
   CTA?: ReactNode;
+  HeadingLeftIcon?: ReactNode;
+  showBackButton?: boolean;
+  // use when content needs to expand with flex
+  flexChildrenContainer?: boolean;
```

Comment:
I asked Peer before making the changes to Shell, I needed to be able to use flex on `TeamAvailabilityScreen`, there were two `div` containers and one `main` container that did not have the correct flex classes preventing me from expanding the contents to fill the remainder of the screen. Adding these classes broke the rest of the app so I decided to make a prop to conditionally add flex to those containers. This is currently only used on the availability screen, but could also be useful in the future.